pep-0682.rst

```diff
@@ -27,9 +27,7 @@ zero to be normalized to positive zero.
 Motivation
 ==========
 
-Here is negative zero:
-
-.. code-block:: pycon
+Here is negative zero::
```

Comment:
to summarize:
  * the console-specific formatting looks great with the new rendering system for peps.python.org
  * however, rendering on the standard pep site is botched.  @AA-Turner notes it may be hard to fix.  (I'd be happy to help if it's feasible.)
  * there is only one other PEP so far using `pycon`
  * it seems we'll need a script to convert console blocks in existing PEP's anyway, so perhaps no need to put this PEP on the bleeding edge

Reply:
interesting-- the new rendering deduces console blocks anyway

(screen shot after merge of this PR, and confirming other edits are reflected)
![Screen Shot 2022-02-11 at 9 28 24 AM](https://user-images.githubusercontent.com/1708631/153519750-814af527-aaab-4c0a-91de-3b04c4536cd9.png)

Reply:
As @AA-Turner explained on the other PR, completely nuking the explict `.. code-block` declaration is not the correct way to fix this. Instead, you should simply change `pycon` to `python` to use the standard Python script file syntax highlighting that doesn't have this problem (which is currently the default, but that is an implementation detail that may change) rather than removing it completely.

> it seems we'll need a script to convert console blocks in existing PEP's anyway, so perhaps no need to put this PEP on the bleeding edge

There's not so much benefit to doing so on existing PEPs that would justify the noise, except on Active/Process PEPs as they are updated, so long as we keep the default `python`, and if not then console blocks are no different than any other non-plain-text code block in that regard.

> interesting-- the new rendering deduces console blocks anyway

Indeed—my testing also seems to confirm there is no difference when rendered via the new system, at least for these particular (relatively simple) blocks.